tests/locationConfigTests.json

```diff
@@ -56,6 +56,16 @@
             "credentialsProfile": "default"
         }
     },
+    "aws-test-mismatch": {
+        "type": "aws_s3",
+        "legacyAwsBehavior": true,
+        "details": {
+            "awsEndpoint": "s3.amazonaws.com",
+            "bucketName": "multitester555",
```

Comment:
please make sure versioning is enabled on this so that healthchecks pass

Reply:
since it's the same account as aws-test one, it should be fine :)